orca_detector/noise_collector.py

```diff
@@ -1,71 +0,0 @@
-# -*- coding: future_fstrings -*-
```

Comment:
Why delete `noise_collector.py`?  We should keep it around for reference since it's the code we used to populate our Noise training data.

Reply:
added back the file, I wanted to allow the same functionality in the live inference function, but this would make the functions unnecessarily complex

Reply:
Agreed.  I'm sure we could merge them, but I don't think it's worth the effort.